src/components/tree-item/tree-item.tsx

```diff
@@ -218,12 +222,16 @@ export class TreeItem
 
   private isSelectionMultiLike: boolean;
 
+  /** Specifies the scale of the tree-item controlled by parent, defaults to m */
+  scale: Scale = "m";
```

Comment:
It looks like the `@Prop` decorator is missing, maybe that's causing issues somehow?

Reply:
I left it out because we didn't want scale on children to be public. Is there a better way to do this?

Reply:
You were right about this causing an issue in that not reflecting it was causing the styling to not apply. I brought the prop back for the time being to address it later in a more relevant issue.